docs/workflows/ios-specific-workflow-steps/xcodebuild-for-devices.md

```diff
@@ -169,4 +169,30 @@ When you need to reduce the verbosity of the `xcodebuild` logs, you can achieve
 end
 ```
 
-Now, the `run_command_simple()` function will execute your customized `xcodebuild` command.
\ No newline at end of file
+Now, the `run_command_simple()` function will execute your customized `xcodebuild` command.
+
+### How can I resolve the OpenSSL signing error?
+
+All tools running in Appcircle cloud environments are controlled by the Appcircle development teams on runners and updated when necessary. One of the tools used on runners is **LibreSSL**. In Appcircle Cloud environments, the **LibreSSL** 3.3.6 version on macOS Sonoma and the **LibreSSL** 2.8.3 version on macOS Monterey are used. For more information, please visit our [**Build Infrastructure**](/infrastructure/ios-build-infrastructure#ios-build-agent-stacks) documentations.
+
+Although **LibreSSL** and **OpenSSL** are alternatives to each other, there are differences between them. **LibreSSL** comes by default with macOS machines and is managed by **Apple**. For this reason, since Appcircle does not have direct access to self-hosted environments, some user-side work on runners can replace **LibreSSL** with **OpenSSL** or update their versions. If **OpenSSL** is used instead of **LibreSSL** for any reason in your self-hosted environments, you will get an **error** like the one below.
+
+:::info Cloud Customers
```

Comment:
IMO, this information should be under the error message. Because the error message should be at the top of the FAQ so users can see whether it fitst their error message or not.